### PR TITLE
Update Generator Name

### PR DESCRIPTION
**Pull request by mikaelkrief**
Thank you for your pull request!
By completing this pull request, you agree to the [Contributing License Agreement](https://github.com/ALM-Rangers/Yeoman-extension-code-generator/blob/master/.github/CLA.md).

**Changes proposed in this pull request:**  
- Update the generator name to : team-services-extension (package.json, tests,index.js)
- update the welcome message
- delete xunit.xml

### CODE REVIEW

**Comment by wpschaub** [comment]
Mikael, can we drop the obj folders in the templates and ignore the vs folder? It "seems" that it's not needed and adds noise.

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
-# Yeoman-extension-code-generator
+﻿# Yeoman-extension-code-generator
```
Comment (willsmythe) [inline README.md]: This L1 should be friendlier, maybe: `Team Services Extension Generator`

File: README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
 >  Team Foundation Server and Team Services extension code generator.
 
-![VSTS](https://almrangers.visualstudio.com/_apis/public/build/definitions/7f3cfb9a-d1cb-4e66-9d36-1af87b906fe9/137/badge) | [![NPM](https://nodei.co/npm/generator-vsts-ext.png?mini=true)](https://www.npmjs.com/package/generator-vsts-ext)
+![VSTS](https://almrangers.visualstudio.com/_apis/public/build/definitions/7f3cfb9a-d1cb-4e66-9d36-1af87b906fe9/137/badge) | [![NPM](https://nodei.co/npm/generator-team-services-extension.png?mini=true)](https://www.npmjs.com/package/generator-team-services-extension)
 
 ## Capabilities
 
-Generator-vsts-ext is [Yeoman](http://yeoman.io/) generator that create an extension for Visual Studio Team Services or Team Foundation Server.
+**generator-team-services-extension** is [Yeoman](http://yeoman.io/) generator that create an extension for Visual Studio Team Services or Team Foundation Server.
```
Comment (willsmythe) [inline README.md]: " .. is **a** Yeoman generator that **creates** an extension .. "

File: package.json [hunk 3]
```diff
@@ -1,78 +1,78 @@
...
-         "mocha": true
-      }
-   },
-   "license": "MIT",
-   "scripts": {
-      "test": "istanbul cover --report cobertura --report html node_modules/mocha/bin/_mocha -- -R xunit-file"
-   }
+  "name": "generator-team-services-extension",
+  "version": "1.0.15",
+  "description": "Team Foundation Server and Team Services extension code generator",
```
Comment (willsmythe) [inline package.json]: `Yeoman generator for Team Services and Team Foundation Server extensions`  ?

**Review by willsmythe — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (wpschaub) [reply README.md]: Done.

File: package.json (hunk 3 above)
Comment (wpschaub) [reply package.json]: Done

File: README.md (hunk 1 above)
Comment (wpschaub) [reply README.md]: Done

**Review by wpschaub — APPROVED** [review]
After making Will's changes I approve.

**Comment by willsmythe** [comment]
Looks good to me!

**Review by willsmythe — APPROVED** [review]

**Pull request merged**